File: mobile/src/main/java/io/ashdavies/cinnamon/home/HomeModule.java
package io.ashdavies.cinnamon.home;

import android.support.v4.app.FragmentActivity;
import dagger.Binds;
import dagger.Module;

@Module
public abstract class HomeModule {

  @Binds
  abstract FragmentActivity activity(HomeActivity activity);

  @Binds
  abstract HomeView view(HomeActivity activity);
}

File: mobile/src/main/kotlin/io/ashdavies/cinnamon/home/HomeModule.kt
package io.ashdavies.cinnamon.home

import android.support.v4.app.FragmentActivity
import dagger.Binds
import dagger.Module

@Module
abstract class HomeModule {

  @Binds
  internal abstract fun activity(activity: HomeActivity): FragmentActivity

  @Binds
  internal abstract fun view(activity: HomeActivity): HomeView
}
